src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Action action;
+    private String fruit;
+    private int value;
+
+    public FruitTransaction(Action action, String fruit, int value) {
```

Comment:
Follow the encapsulation principle. Consider making the members of the class private and provide getters and setters to access them if necessary.